cmake/GetVulkan.cmake

```diff
@@ -0,0 +1,79 @@
+find_package(PkgConfig REQUIRED)
+include(ExternalProject)
+
+function(clearVars)
+    set(CHECK_VARS GLSLANG_VALIDATOR_EXECUTABLE;INCLUDE_DIR;LIBRARY;GLSLC_EXECUTABLE;CFLAGS;INCLUDEDIR;INCLUDE_DIRS;LDFLAGS;LIBDIR;LIBRARIES;LIBRARY_DIRS;MODULE_NAME;PREFIX;STATIC_CFLAGS;STATIC_INCLUDE_DIRS;STATIC_LDFLAGS;STATIC_LIBRARIES;STATIC_LIBRARY_DIRS;VERSION;Headers_FOUND)
+    foreach(ITEM IN LISTS CHECK_VARS)
+        if(Vulkan_${ITEM})
+            unset(Vulkan_${ITEM} CACHE)
+            unset(Vulkan_${ITEM} PARENT_SCOPE)
+        elseif (Vulkan${ITEM})
+            unset(Vulkan${ITEM} CACHE)
+            unset(Vulkan${ITEM} PARENT_SCOPE)
+        endif()
+    endforeach()
+endfunction()
+
+if(NOT Vulkan_FOUND)
+    message(STATUS "Checking for module 'vulkan' > 1.3.255")
+    find_package(Vulkan REQUIRED)
```

Comment:
This should be
`find_package(Vulkan 1.3.255 QUIET)`
Having REQUIRED will cause it to fail immediately and never get to the `pkg_check_modules` version